AOloopControl/examples/scexao-vispyr-bin2-conf/simLHS/respM.fits.get.bash

```diff
@@ -6,9 +6,9 @@ if [ ! -f $FILENAME ]; then
 
     ggID='1MnNo9WfRHuKik3gpEHEjZHgt8_PM0T1a'
     ggURL='https://drive.google.com/uc?export=download'
-    filename="$(curl -sc /tmp/gcokie "${ggURL}&id=${ggID}" | grep -o '="uc-name.*</span>' | sed 's/.*">//;s/<.a> .*//')"
-    getcode="$(awk '/_warning_/ {print $NF}' /tmp/gcokie)"
-    curl -Lb /tmp/gcokie "${ggURL}&confirm=${getcode}&id=${ggID}" -o "${filename}"
+    filename="$(curl -sc /tmp/gcookie "${ggURL}&id=${ggID}" | grep -o '="uc-name.*</span>' | sed 's/.*">//;s/<.a> .*//')"
+    getcode="$(awk '/_warning_/ {print $NF}' /tmp/gcookie)" || touch /tmp/gcookie
+    curl -Lb /tmp/gcookie "${ggURL}&confirm=${getcode}&id=${ggID}" -o "${filename}"
     mv scexao-vispyr-bin2-zrespM.fits ${FILENAME}
```

Comment:
This line is unnecessary if the end of the curl command that transfers the file ends with ```-o "${FILENAME}"```, although I understand if we want to cache the curl result first and then ```mv``` it.